projects/step-core/src/lib/modules/basics/services/auth.service.ts

```diff
@@ -98,11 +98,14 @@ export class AuthService implements OnDestroy {
   }
 
   goToLoginPage(): any {
```

Comment:
can the return type be void? if not, nevermind

Reply:
because it looks like inside there is navigation involved, even tho it is returned

Reply:
Since we never use the return value anyway I don't see the reason, so void it is.